answer_files/2008_r2_core/Autounattend.xml

```diff
@@ -49,8 +49,8 @@
                     <InstallToAvailablePartition>false</InstallToAvailablePartition>
                     <InstallFrom>
                         <MetaData wcm:action="add">
-                            <Key>/IMAGE/NAME</Key>
-                            <Value>Windows Server 2008 R2 SERVERSTANDARDCORE</Value>
+                            <Key>/image/index</Key>
+                            <Value>1</Value>
```

Comment:
This index seems wrong as we use the identical ISO for both the desktop and the core edition. I haven't looked through all the other modified templates.